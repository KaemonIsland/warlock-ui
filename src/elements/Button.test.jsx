import renderer from 'react-test-renderer'
import React from 'react'
import { Button, ThemeProvider } from '..'

// prettier-ignore
test('The Button works!', () => {
  const tree = renderer
    .create(
      <ThemeProvider>
        <Button title="Click me" callback={() => {}} />
      </ThemeProvider>,
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
