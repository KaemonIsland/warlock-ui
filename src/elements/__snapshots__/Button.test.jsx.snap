// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Button works! 1`] = `
<button
  className="Button__StyledButton-sc-4zekuf-0 jFyzKS"
  onClick={[Function]}
>
  Click me
</button>
`;
